vn.hoabinh.quan.CoBang.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: vn.hoabinh.quan.CoBang
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: cobang
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
+  - --device=all
+
+modules:
+
+  - flatpak/libzbar.yml
```

Comment:
It's better to avoid splitting the modules into separate files unless they are generated automatically somehow

Reply:
Ok, will merge them. I just look into other projects as reference and thought that it is good practice.